iosApp/PhongKMMIC/Sources/Presentation/Coordinator/Screen.swift

```diff
@@ -6,10 +6,12 @@
 //  Copyright © 2023 Nimble. All rights reserved.
 //
 
+import shared
+
 enum Screen {
 
     case splash
     case login
     case home
-    case surveyDetail
+    case surveyDetail(survey: SurveyUiModel)
```

Comment:
Sending `UiModel` here is not a good practice. We should send a raw model (Survey) or only surveyId (string)

Reply:
Solved in ecb0b79